asn1c_combined/doIt.sh

```diff
@@ -1,15 +1,28 @@
 #!/bin/bash
 
+# This script assumes that the J2735 files have already been generated.
+# To generate the J2735 files, run generate-files.sh with asn1c installed.
+
 export LD_LIBRARY_PATH=/usr/local/lib
 export CC=gcc
 
-asn1c -fcompound-names -gen-PER -gen-OER -pdu=all \
```

Comment:
Is there a reason prefer to manually copy the generated source cpp and header files to automating this step during the docker build?

Reply:
According to the header comments in the ASN J2735 2020 files, we are not permitted to redistribute the ASN J2735 2020 files. Due to this, we removed the ASN J2735 2020 files from source control and deigned to have the developer retrieve the files themselves.

Once we removed the ASN files, the CI checks were failing because the automated step was attempting to use files that weren't there.

Moving to manually copying the generated source cpp & header files allowed us to keep the ASN files out of source control and it allowed the CI check to pass.

An additional benefit is that the build time has been reduced.